app/javascript/pages/form/components/YesNoRadio.tsx

```diff
@@ -1,17 +1,46 @@
 import React from "react"
-import { t } from "@bloom-housing/ui-components"
-import { Card } from "@bloom-housing/ui-seeds"
+import { t, FieldGroup } from "@bloom-housing/ui-components"
+import { useFormStepContext } from "../../../formEngine/formStepContext"
+import "./YesNoRadio.scss"
 
 interface YesNoRadioProps {
-  label: string
+  label?: string
+  note?: string
+  description?: string
+  yesText?: string
+  fieldNames?: {
+    question?: string
+  }
 }
 
-const YesNoRadio = ({ label }: YesNoRadioProps) => {
+const YesNoRadio = ({ label, note, description, yesText, fieldNames }: YesNoRadioProps) => {
+  const { register, errors, watch } = useFormStepContext()
+  const selected = watch(fieldNames?.question)
   return (
-    <Card>
-      <Card.Header>YesNoRadio Component</Card.Header>
-      <Card.Section>{t(label)}</Card.Section>
-    </Card>
+    <>
+      {t(label)}
```

Comment:
Will this just be a bare string?  We wouldn't want to use the groupLabel prop for the label?